### ISSUE TITLE
SelectWrapper: broken if used with 200% text only zoom in Firefox

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#2050]
Description  
---
In Firefox with 200% text only zoom enabled, the SelectWrapper component does not scale correctly. This is caused by a fixed height which is set in pixel on the inner `button` element.

To Reproduce
---
1. In **Firefox** go to https://designsystem.porsche.com/latest/components/select/examples 
1. Zoom up text only to 200%
1. See false scaling of select component

Expected behavior 
---
The SelectWrapper component should support text only zooming until 200% without any visual and functional issues.

Note
---
- Fixed values should be changed to relative values to also support other browsers

Screenshots
---
<img width="557" alt="Bildschirmfoto 2022-09-15 um 08 45 30" src="https://user-images.githubusercontent.com/40788023/190335091-05f67434-b4ca-46be-bfce-e7be1fa9f210.png">

Test Environment
---
Platform/Browser/Version: [macOS Firefox]
